### PR TITLE
Miningbase Rework

### PR DESCRIPTION
**Pull request by Lovebird-Frungy**
# General Documentation

### Intent of your Pull Request

Remakes the entire mining base in order to bring a better playing experience to mining, with (vacant) room(s) for improvement, adapting to further content.

Overall its been pretty fun to work on this project (this is the sixth time i do it all over again), and i’ll understand if people prefer to stick to the old one.

### Why is this change good for the game?

The changes are made to address two issues:
-The mining base being an experimental facility on a deadly planet, it needed to be tweaked a bit more and security amped up, that is where the reworked atmos, closer medical facility, blast doors and secondary exits come in, the base being often dented, it needs to be built to last, the current mining base does not have that much resilience.
-The mining base is the place some players will spend most of their shift, and as we’re shifting into a more rp server, i felt it was necessary to make it a better place to live in order to facilitate interactions between players, that's where the vacant room, closer break room, and reworked layout come in. You no longer have to go to the other end of the base to see if other miners are chilling.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Miningbase has been revamped, layout completely redone and new areas added.
Now the areas are:
Two EVAS
Living area with internal shuttle dock
Engineering (Tcomms, atmos and power)
Vacant Room
Vacant Maintenance
Break Room
Infirmary
Processing (in case the orm blew up, a relic of times of yore that will probably be removed soon and then the room will be repurposed)

New mining base picture:
NuMining_Love

With areas:
NuMining_Love_Areas

New features:
Internal shuttle dock, pretty and safe, with railings to prevent accidental gibbing.
Infirmary directly linked to EVA.
A secondary EVA and an emergency exit.
Personal miner equipment lockers with gps.
Anchored lockers so stuff doesn’t fly out during breaches, making the place nicer to look overall.
Mining lock down system with blast doors
Bunk dormitories (overall the only downgrade of the pr, the dorms are not private anymore, but i feel it is fitting rp-wise as life ain’t cheap and this is a base of operation, not a luxury hotel)
Curtained windows to prevent local fauna from being agitated too much.
Reworked gas mining, fully functional and allowing mining to be self sufficient atmos-wise.
Security post.
Disposals leading to lava for trash disposing.
Cryopods.
Mining medic locker now has binoculars (in order to stay away from megafauna), and earmuffs (for gibtonite ear damage healing), probably should add this to the base locker content but might need some input.
Bigger external access for less clogging.

Mining shuttle has been remade in order to adapt to the new layout, all the while being compatible with other stations.

New mining shuttle picture:
image

New area have been added, old areas renamed and new area sprites done for easier future mapping.

### What should players be aware of when it comes to the changes your PR is implementing?

They now have a locker where they can find their gps and stash their loot.
A portable air pump can be used to overcharge their oxygen tanks.
New vacant room for them to use their imagination.
Auxiliary EVA to facilitate work.

### What general grouping does this PR fall under? 

Mining Mapping.

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No.

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

No.

# Changelog

:cl:  Lovebird
rscadd: Shiny new miningbase
rscdel: Removed the old miningbase
imageadd: Added a few area icons for mapping
/:cl:

### CODE REVIEW

**Comment by Tipy1802** [comment]
The current one is better. The more recreational rooms are on the back and a bit seperated because they can be used to exile people to (like heads of staff during revs)

**Comment by Bibby0110** [comment]
It’s very compact compared to the old one, I have to agree with tipy on this.

**Comment by ToasterBiome** [comment]
I like the idea, it does seem very dense though. and it has a boring shape, old mining base reminded me of a building they would just constantly and haphazardly expand like it’s this clunky shitty outpost on a deadly planet (which is actually what happened) Maybe you can start with the old mining base and expand it/add on? A revamped mining base could have the shuttle protected in more of a U shape so a certain bubblegum has a harder time coming up to the station. I like it either way, would at least like to see a testmerge so miners could see how they like it

**Comment by athena413** [comment]
THIS is peak template right here

thank you for making wiki life easier

**Comment by Moggram** [comment]
Ummmm... what is going on with the top right room? Is it for engs to play round and build stuff?

**Comment by ToasterBiome** [comment]
> Ummmm... what is going on with the top right room? Is it for engs to play round and build stuff?

I assume this is a spare room for further expansion

**Comment by athena413** [comment]
THINGS I LOVE:

The circular design! Mining felt, previously, like two entirely seperate halves connected by a shitty tube that broke half the time. This feels more unified

Two exits. More options for directions miners can go and come in from. verigud

I like the empty rooms, potential for engineers to expand and make additional useful things is always quite nice,

Shuttle in the center, so it doesnt get *immediately* smacked by whatever gets brought to the base

THINGS I DON'T:

Center docking area needs some clutter, it looks weird.

You really should test what happens if you undock the mining shuttle from the base, having it land on plating might be asking for trouble

Bring the tables back to the shuttle!

Please give the north entrance of the base some stuff, it looks terribly empty

Switch the bathroom rooms around! Toilets first, then another door to get to a shower. It makes more sense that way.

**Comment by ChesterTheCheesy** [comment]
Could do for some posters inside the base, and maybe even inside the mining ship. Miner job is stressful, so they would fit nicely.

**Comment by Lovebird-Frungy** [comment]
> The current one is better. The more recreational rooms are on the back and a bit seperated because they can be used to exile people to (like heads of staff during revs)

I understand the old design with two areas separated by a bridge has its merits, but in my own experience i find it pretty annoying.

> It’s very compact compared to the old one, I have to agree with tipy on this.

The design philosophy was to make a compact base with optimized space, so i guess it is normal you feel that way. 

> I like the idea, it does seem very dense though. and it has a boring shape, old mining base reminded me of a building they would just constantly and haphazardly expand like it’s this clunky shitty outpost on a deadly planet (which is actually what happened) Maybe you can start with the old mining base and expand it/add on? A revamped mining base could have the shuttle protected in more of a U shape so a certain bubblegum has a harder time coming up to the station. I like it either way, would at least like to see a testmerge so miners could see how they like it

Thats a way to put it, i must admit it didn't went that way when i redesigned it.
The first designs were indeed made to resemble the old base but i ended up just discarding it and making something brand new.
I've had some input from mining players which liked it, but we will see.
If it isn't kept, i might just upgrade the current base, although i don't really like it myself, the people have the final word.

> THIS is peak template right here
> 
> thank you for making wiki life easier

I can't tell if this is sincere or sarcastic, either way i made this map again and again so many times i can tell every new features by looking at the picture.

> Ummmm... what is going on with the top right room? Is it for engs to play round and build stuff?

Yes, exactly, the room is literally named "Vacant Room" and is here to be used in all kinds of silly projects.

> THINGS I LOVE:
> 
> The circular design! Mining felt, previously, like two entirely seperate halves connected by a shitty tube that broke half the time. This feels more unified
> 
> Two exits. More options for directions miners can go and come in from. verigud
> 
> I like the empty rooms, potential for engineers to expand and make additional useful things is always quite nice,
> 
> Shuttle in the center, so it doesnt get _immediately_ smacked by whatever gets brought to the base
> 
> THINGS I DON'T:
> 
> Center docking area needs some clutter, it looks weird.
> 
> You really should test what happens if you undock the mining shuttle from the base, having it land on plating might be asking for trouble
> 
> Bring the tables back to the shuttle!
> 
> Please give the north entrance of the base some stuff, it looks terribly empty
> 
> Switch the bathroom rooms around! Toilets first, then another door to get to a shower. It makes more sense that way.

Well, here we see that they are two views on the "two halves" design of the old base, again, we will see what players like most.
The two exits idea was from wej.

Center docking was designed to be a walking area, not one people are supposed to stay for long, but i see what you mean.
The shuttle is working, i tested it many and many times.
I personnaly hate the shuttle tables, as i don't think a space vessel should have unsecured equipment.
North entrance is indeed pretty barren, i do not really know what to place there, i just kinda went for the philosophy that the miner will place their stuff there if needed, so i didn't place anything to make it look clean.
UUUUUUUH, i kinda don't want someone to go through me having my private break so he can have a shower, also this is the same sequencing as in the old base, but maybe people would prefer it the way you described, it just sounds wrong to me.

> Could do for some posters inside the base, and maybe even inside the mining ship. Miner job is stressful, so they would fit nicely.

Great idea, i completely forgot this kind of decoration, will think about it once i have more input.


To note we could also go the midway and make the miningbase random between the old design and the new, but i'm unsure as it might really disorient players.

**Comment by athena413** [comment]
> To note we could also go the midway and make the miningbase random between the old design and the new, but i'm unsure as it might really disorient players.

miners hate inconsistency, personally i would love a system where mining base layouts are based on map (so delta, eclipse can have larger mining bases fit for many many miners) but thats something for another time

**Comment by Moggram** [comment]
>
>
>Great idea, i completely forgot this kind of decoration, will think about it once i have more input.

You could also add some dirty tiles, piece of ore, piper tray, random tools and potted plants round the place. Maybe even a wreck/half made Ripley, but that could be a bit too much. I think it will give it a more lived in feel.

**Comment by patpol4** [comment]
Is this tested for lighting/airlock cycling length/atmos sufficiency?

We could testmerge it for some time to see what people think about it or just implement this as a template somehow.

Also it would probably be good to get some more miner opinions on this.

**Comment by Lovebird-Frungy** [comment]
> 
> 
> Is this tested for lighting/airlock cycling length/atmos sufficiency?
> 
> We could testmerge it for some time to see what people think about it or just implement this as a template somehow.
> 
> Also it would probably be good to get some more miner opinions on this.

Yes, it is playtested and working perfectly.
Some real server testing would be really helpful. I’ve had input from a enthusiastic miner already and it helped a lot.

**Comment by Cartlord** [comment]
Shaft Miner here, 56.2 hours. 
I don't see the point of adding a cryosleep pod to the mining base, both in gameplay and logical terms - I don't see any reason anyone would want to cryo and not be bothered to just go to dorms, and I don't see why IC you'd want to go to cryosleep planetside because of how much of a hellhole it is. 
I appreciate the second airlock for the inevitability that one airlock gets busted, but I somewhat question the sheer amount of extra gear now available planetside since it seems like now there's nothing stopping a mining medic from just becoming a worse shaft miner (I've seen it happen, it sucks). 
I question how the shuttle landing in the building works - does that mean it has no roof? If so, what stops the place from being affected by ash storms?
I want to elaborate on a few points with the airlocks - I like how they don't have ANY rock around them, because I found that it was an occasional issue with the current base that you'd have GIBTONITE spawn in the rock right next to the only exit airlock, and if it blew you could only get out of the base properly (because the airlock got weird when breached) by KA'ing a hole in the west window. Speaking of airlocks being weird, I want to ask: will this mining base's airlocks have the same issue that was present in the original's airlocks, where if the airlock's walls are braeched and the air leaks out, the airlock will not repressurize even after the walls are fixed?
I REALLY DON'T LIKE the existence of a Sec checkpoint in the base, because Sec never goes planetside (hell, _nobody_ who isn't a miner goes there) - so any Miner who's an antag, or even just wants to grief, can KA that secpoint open, KA the locker open, and now they have Sec gear and nobody's likely to know besides the other miners.

**Comment by athena413** [comment]
> I REALLY DON'T LIKE the existence of a Sec checkpoint in the base, because Sec never goes planetside (hell, _nobody_ who isn't a miner goes there) - so any Miner who's an antag, or even just wants to grief, can KA that secpoint open, KA the locker open, and now they have Sec gear and nobody's likely to know besides the other miners.

the checkpoint does not have a locker

> I can't tell if this is sincere or sarcastic, either way i made this map again and again so many times i can tell every new features by looking at the picture.

that was genuine - you have no idea how many people just delete the template and dont document their PRs at all

**Comment by Lovebird-Frungy** [comment]
> Shaft Miner here, 56.2 hours.
> I don't see the point of adding a cryosleep pod to the mining base, both in gameplay and logical terms - I don't see any reason anyone would want to cryo and not be bothered to just go to dorms, and I don't see why IC you'd want to go to cryosleep planetside because of how much of a hellhole it is.
> I appreciate the second airlock for the inevitability that one airlock gets busted, but I somewhat question the sheer amount of extra gear now available planetside since it seems like now there's nothing stopping a mining medic from just becoming a worse shaft miner (I've seen it happen, it sucks).
> I question how the shuttle landing in the building works - does that mean it has no roof? If so, what stops the place from being affected by ash storms?
> I want to elaborate on a few points with the airlocks - I like how they don't have ANY rock around them, because I found that it was an occasional issue with the current base that you'd have GIBTONITE spawn in the rock right next to the only exit airlock, and if it blew you could only get out of the base properly (because the airlock got weird when breached) by KA'ing a hole in the west window. Speaking of airlocks being weird, I want to ask: will this mining base's airlocks have the same issue that was present in the original's airlocks, where if the airlock's walls are braeched and the air leaks out, the airlock will not repressurize even after the walls are fixed?
> I REALLY DON'T LIKE the existence of a Sec checkpoint in the base, because Sec never goes planetside (hell, _nobody_ who isn't a miner goes there) - so any Miner who's an antag, or even just wants to grief, can KA that secpoint open, KA the locker open, and now they have Sec gear and nobody's likely to know besides the other miners.

I figured i'd place some cryopods since there was some room and you never know if a miner has to go and decides to let another player take control.
I understand the nap in a deadly planet part but i feel this is more convenient for the occasional disconnecting miner.

The only "mining" gear i've added is a fourth suit storage unit for the sake of symetry. I guess the mining medics will probably take it instead of its intended purpose of being here for the optional extra mining volunteer.

The docking site is affected by ash storms but not the rest of the areas, my idea was of a shutter system to let the shuttle in and out, but i guess its kinda weird how it is working right now.

Sadly, this issue is inherent with the advanced airlock controllers themselves, and your only option is waiting and skipping pressurization. Once the walls are fixed it will work again unless something went wrong in the atmos area.

I've added a sec checkpoint for the cargo guard, giving them a room for themselves, i myself occasionaly go down on mining to check on the miner and talk with them about the dangers of bringing dangerous stuff on station. As for the locker, i anticipated this problem and did not place any, as the cargo guard already has a locker on station. I guess the secdrobe wasn't really necessary however, and i'll probably remove it as there is already one on station too.

**Comment by Cartlord** [comment]
Actually, the issue I was referring to, is that sometimes the old lavaland base airlock STILL won't repressurize even AFTER the windows are replaced and the walls are fixed - no clue why, and I am wondering if a similar issue will exist with this.

**Comment by Moggram** [comment]
> 
> 
> Actually, the issue I was referring to, is that sometimes the old lavaland base airlock STILL won't repressurize even AFTER the windows are replaced and the walls are fixed - no clue why, and I am wondering if a similar issue will exist with this.

Ya that is a atmo thing and can happen on the station to. You just see it more on lavaland because you use the airlocks alot more.

**Comment by ghost** [comment]
I really like this set up, this should be test merged to give it a go.

**Review by patpol4 — CHANGES_REQUESTED** [review]
Fixed some things for you.

Added Frank
Added tables back to shuttle, 
added more lights
dreamseeker_tuH4eaAGRr
made morgue have a bit more space
vivaldi_9P0UzO7zdF
Fixed some active turfs, 
fixed the apc and airlock controller offsets
added a lot of decals to make it a bit less bland
changed some pipes to be visible or invisible
added some crates
fixed this stinky error
vivaldi_W6mPGj9d3d

Anyway, I'll testmerge this but there is still some work to be done, mainly on this room, 
dreamseeker_6GNSzALB7J
its too clean for an abandoned one, if you want it to be sort of an empty room then just block it off with a barricade and add dirt and broken stuff inside, if not then do something else with it, it just can't stay as it is

**Comment by Lovebird-Frungy** [comment]
> Fixed some things for you.
> 
> Added Frank
> Added tables back to shuttle,
> added more lights
> dreamseeker_tuH4eaAGRr
> made morgue have a bit more space
> vivaldi_9P0UzO7zdF
> Fixed some active turfs,
> fixed the apc and airlock controller offsets
> added a lot of decals to make it a bit less bland
> changed some pipes to be visible or invisible
> added some crates
> fixed this stinky error
> vivaldi_W6mPGj9d3d
> 
> Anyway, I'll testmerge this but there is still some work to be done, mainly on this room,
> dreamseeker_6GNSzALB7J
> its too clean for an abandoned one, if you want it to be sort of an empty room then just block it off with a barricade and add dirt and broken stuff inside, if not then do something else with it, it just can't stay as it is

Wow, thanks for the work, i just realized how many mistakes i made:
I completely forgot about frank.
I'm not a fan of tables in the shuttle but i guess it'll have to do.
Lights are good.
Thats a way better morgue, i never thought about it like that. However i'm not sure you can access the curtain anymore.
I have no idea what active turfs are.
I didn't notice that mistake.
Decals are nice, i thought i added enough but i'm clearly not experienced in this area.
Not a big fan of crates everywhere like in the old base.
Stinkystinky indeed.

As for the room, i'm not sure what do do with it. I really wanted it to be like that, a clean slate room that doesn't have to be cleaned before using it for various projects. Its intended as a vacant room, not an abandoned one.

**Comment by UselessTheremin** [comment]
lmoa @grimm-hollowknight bro i wonder if these area sprites are up to the Yoggers Standard :tm: , art like this is really important to the game! i dont go a single second without seeing area textures

**Comment by N3D6** [comment]
i like this idea aswell as the design but i think the old one is better, the layout on this one isn't as great compared to it because its way too compact contrasted to the old one which i think is just fine as is. i'd love to see the two bases alternate between rounds but idk if thats possible or a tall task.

**Comment by nmajask** [comment]
I like the design and the extra room is very nice if you get an R&D console and having a bar of soap in the bathroom is so useful, as on the normal base I need to steal the space cleaner from medical. I just have a few issues with the medical room:

1. There is no good place for you to hang a defib wall mount, as the sign blocks the best spot for it
2. There is no IV drip
3. Its not really an issue, but please put a crew pinpointer in the mining medics locker

**Comment by alexkar598** [comment]
Where are muh holopads

**Comment by Moggram** [comment]
> 
> 
> Where are muh holopads

holopads? For what? AI's don't know how to jump camera networks.

**Comment by Lovebird-Frungy** [comment]
> I like the design and the extra room is very nice if you get an R&D console and having a bar of soap in the bathroom is so useful, as on the normal base I need to steal the space cleaner from medical. I just have a few issues with the medical room:
> 
> 1. There is no good place for you to hang a defib wall mount, as the sign blocks the best spot for it
> 2. There is no IV drip
> 3. Its not really an issue, but please put a crew pinpointer in the mining medics locker

Thanks for the input!
I did not think about defib wall mount, i'll see what i can do.
There is an iv drip in the blood crate.
Good idea! I'll probably edit the mining medic locker contents at this point (i've already added earmuffs and binoculars).



> Where are muh holopads

Thank you! I completely forgot about those.

**Review by nesisknesis — APPROVED** [review]

**Comment by Cartlord** [comment]
I notice that having an air pump instead of an air can means you can now use that to fill an air tank to over 2000 kPa, much more than you previously could. I don't mind this, but figured it was worth pointing out.

**Comment by Lovebird-Frungy** [comment]
> I notice that having an air pump instead of an air can means you can now use that to fill an air tank to over 2000 kPa, much more than you previously could. I don't mind this, but figured it was worth pointing out.

Well, it was intentional, i guess it should be added in the wiki if the base gets merged.
However it may be unbalanced, so further input is always welcome.

**Comment by ghost** [comment]
Haven't done a lot shaft mining myself, but to be honest I am not a big fan of the circular shape of the base. On the other hand, it's nice to have a new look soooo I am not sure where I stand.

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm [hunk 1]
```diff
@@ -135,6 +135,9 @@
 	new /obj/item/clothing/mask/gas/explorer(src)
 	new /obj/item/storage/belt/medical(src)
 	new /obj/item/pickaxe(src)
+	new /obj/item/twohanded/binoculars
```
Comment (SomeguyManperson) [inline code/game/objects/structures/crates_lockers/closets/secure/medical.dm]:
```suggestion
	new /obj/item/twohanded/binoculars(src)
```
need to do this for all the new added stuff

**Comment by patpol4** [comment]
Will see about making an ingame poll to get more player opinion on whether we keep the old one, replace, or do a rotation thing.

File: code/game/objects/structures/crates_lockers/closets/secure/medical.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/structures/crates_lockers/closets/secure/medical.dm]: Oh, thanks, i am still new to this stuff.

**Comment by Lovebird-Frungy** [comment]
> Haven't done a lot shaft mining myself, but to be honest I am not a big fan of the circular shape of the base. On the other hand, it's nice to have a new look soooo I am not sure where I stand.

Yeah, you share the same critic as a lot of other people.
I must say that this design is heavily influenced by my own preferences, i wanted to make something "generic", symetric, and efficient, but i took away the "haphazard" charm of the old base.
It is a fundamental flaw in the design, and i hoped at least some people would agree with my vision.
Albeit i had to make some concessions from the earlier designs (you poor people will never experience the mining picnic dome).
Anyways, thanks for the feedback.

**Comment by Cartlord** [comment]
First-off, I wanna say, I'm very interested in this "mining picnic done"; second-off, I've noticed that the increased amount of machinery makes it so that it takes more power from the SMES, so now there's actual reason to use the PACMAN generator. It also means there's more reason for an Engineer to upgrade the machines at the minebase (especially the PACMAN) - which, in my experience, simply does not happen.
Personally, I see this as good and bad - I like Engineers having more interaction with the minebase, but I hate that most of them just DON'T upgrade anything there, because they didnt get the memo that there's any reason to.
I like the new design, but I don't feel it fits Yogs due to the simple apathy of Engineering regarding it.

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 92e694acd1b4ddb5bdab7f42dbf75d255058fcd2
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: da56d524b85eeabf513b808c3ed6a8c9df605f9a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 6eac6b8678728327d8878fa07e5c0f4655a8fa3e
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 92b18435786db50501d15b57a7d0f399158dcce7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 6eac6b8678728327d8878fa07e5c0f4655a8fa3e
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 4cf045dcc80de3751ed501a3fdf71a161b4b6078

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 09dc7735c329443b1d7140a2d32a9b7e5388298f
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 8ce5531e4aa90f730f2881ca15125742069cc3b7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 50e22d907e88fc4aed08712ae4939d0ca006b9bd
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: cfbef971e39f756f57c6351ccba16176fb7ca9b7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: d244ed8af6b4625f69aff980ca6f669c8eab96cc
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: c9dd238bda1b13104771876231c7f061360f117a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: b34fc836328b6ea784801f980d65536c09daad58
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 99a8136a2c3ea2386b7417cadce313eef7205037

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 51d15f34c48d85520d7724f0b4ab5cd650e8be05
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 08a5b01226a862a638855658a1fcb349f2180bdf

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: faf404bfec32c473db5ed3f6fa8061fcaa8b01d8
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 261ac2b415670d9615de140a821f0aed25f45412

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: dfdf9d25fba679e22d2ccaca56dc3ff10960e5af
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: a57b89a5cefc62a3059460567fc50d2b23efa5f5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: dfdf9d25fba679e22d2ccaca56dc3ff10960e5af
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: d723368c137b9977e75b3e26dfb321c788297cdd

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 24d1f239ea0aa6c88caabde6243b7528fe97fb4c
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 1c84e2f8d3d922d24bcc92d7edb9ce9e32d5252e

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: e581b726a44943425eb38bb248e800e792d0fd9d
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: fc7122cf9f35688fd96d1b459cc1c9b61682e752

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 1befeb33968cd7fcd11341f081e90e149979098b
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 87f7ce98fbe6e2119e87704ba716dc7213e89bd9

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: bfcf8bfb1e922f1d6c8f6db19948a76bbfa39116
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: e56275b82d8a1b0a3a432a233629ac206864f83a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 074dc717e2c59a75f46c232f5d068130b857d36d
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: e10404c9a10c2ee7f917b59885fa18ede7f0435a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: aeb34a71e3fd3563bcfadbc47b1b7574c1f9359b
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 3fc96da3fe7b326106f45bec9fcd29d2afa4b3c4

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 5b96721222519db28316c53f605c7f8603965bfc
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 0e5c469765b215fb6c9c7efe7e82d2ab360f0212

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: a2b0d80bb2fe24c63b5611d324b01f653679b0fa
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 48ff5736e8e42a68fceb7e361203ebf0fa1bbeb2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 75e241882ae3eba86957aa8ef47647e805b2a28b
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: a573d4fda4232a30130441438b8607ed178df93e

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: b0848cac60e6718660d3154778f47593877aaf0e
Pull Request: 383f729aa094342e4d9215d75a57df0c4792b5ee
Server: 46c827dc9eeae1ddda6382961b987de97833563a

**Comment by adamsong** [comment]
Poll puts the new layout above the old one, @wejengin2 not sure what you want to do with this.

**Comment by patpol4** [comment]
> Poll puts the new layout above the old one, @wejengin2 not sure what you want to do with this.

The poll has spoken, I'll review this once more with lovebird soon and then fullmerge.

**Comment by Lovebird-Frungy** [comment]
> First-off, I wanna say, I'm very interested in this "mining picnic done"; second-off, I've noticed that the increased amount of machinery makes it so that it takes more power from the SMES, so now there's actual reason to use the PACMAN generator. It also means there's more reason for an Engineer to upgrade the machines at the minebase (especially the PACMAN) - which, in my experience, simply does not happen.
> Personally, I see this as good and bad - I like Engineers having more interaction with the minebase, but I hate that most of them just DON'T upgrade anything there, because they didnt get the memo that there's any reason to.
> I like the new design, but I don't feel it fits Yogs due to the simple apathy of Engineering regarding it.

Well, it was an earlier design when wej told me they missed the old bridge, so i made this:
image

The idea was to make it a barbecue: picnic table, grill, "kiss the cook" apron. But i never got it right and i moved to a new general design.

For the power issue, i understand what you mean, and its a shame non miners often ignore the place, even half of the qms don't care about it either. However the pacman doesn't have to be upgraded to provide enough power for the whole base.
I thought about giving mining its own engine, but it has its own lot of problem.

**Comment by Cartlord** [comment]
I can't get the coinpress to work at all, though that may just be an issue with the press itself.

**Comment by ynot01** [comment]
very happy with this layout personally after playing with it for so long, only problem is that nigh every single locker and crate is bolted to the floor roundstart, this makes it quite frustrating when a wrench is not readily on person and you want to transport some things

**Comment by Lovebird-Frungy** [comment]
> I can't get the coinpress to work at all, though that may just be an issue with the press itself.

Maybe indeed, i made sure to get the vars like in the original, i think the press code might need some attention.

> very happy with this layout personally after playing with it for so long, only problem is that nigh every single locker and crate is bolted to the floor roundstart, this makes it quite frustrating when a wrench is not readily on person and you want to transport some things

Repeating what i said on discord, it is intended, albeit it is a personal preference. Given that the base tends to get breached, i made sure the equipment is secured, its also the reason why there are so few items just lying around.

**Comment by patpol4** [comment]
Its better if its secured given the locker positioning, if you really want to move it use the wrench

**Comment by Cartlord** [comment]
FOr the record there's a wrench in the base, in a toolbox in the TComms room

**Review by patpol4 — APPROVED** [review]
After months of testmerge i believe this is good to go, will merge later today.
If you have any last comments, now's the time.

**Comment by patpol4** [comment]
Let's go then
